### PR TITLE
fix: fix remaining JSDoc

### PR DESCRIPTION
**Pull request by miyuesc**

### CODE REVIEW

File: lib/util/RenderUtil.js [hunk 1]
```diff
@@ -67,6 +67,11 @@ function curveTo(p1, p2, p3) {
   return [ 'C', p1.x, p1.y, p2.x, p2.y, p3.x, p3.y ];
 }
 
+/**
+ * @param {Point[]}waypoints
```
Comment (nikku) [inline lib/util/RenderUtil.js:71]:
```suggestion
 * @param {Point[]} waypoints
```

File: lib/util/RenderUtil.js [hunk 2]
```diff
@@ -67,6 +67,11 @@ function curveTo(p1, p2, p3) {
...
+ * @param {number}[cornerRadius]
```
Comment (nikku) [inline lib/util/RenderUtil.js:72]:
```suggestion
* @param {number} [cornerRadius]
```

Comment (miyuesc) [reply lib/util/RenderUtil.js]: Sorry, it was my mistake.

File: lib/features/keyboard/Keyboard.js [hunk 3]
```diff
@@ -182,7 +190,7 @@ Keyboard.prototype._fire = function(event) {
  *
  * @param {number} [priority]
  * @param {Function} listener
- * @param {string} type
+ * @param {string} [type = 'keyboard.keydown']
```
Comment (nikku) [inline lib/features/keyboard/Keyboard.js:193]:
```suggestion
 * @param {string} [type='keyboard.keydown']
```

File: lib/features/keyboard/Keyboard.js [hunk 4]
```diff
@@ -194,6 +202,12 @@ Keyboard.prototype.addListener = function(priority, listener, type) {
   this._eventBus.on(type || KEYDOWN_EVENT, priority, listener);
 };
 
+/**
+ * Remove a listener function.
+ *
+ * @param {Function} listener
+ * @param {string} [type = 'keyboard.keydown']
```
Comment (nikku) [inline lib/features/keyboard/Keyboard.js:209]:
```suggestion
 * @param {string} [type='keyboard.keydown']
```

**Review by nikku — COMMENTED** [review]
Great stuff!

**Pull request merged**